### ISSUE TITLE
Error on upgrading amber from 0.11.1 to 0.27.0

### ISSUE THREAD

**Issue by zubroid** [amberframework/homebrew-amber#2]
Hi! 

I'm having an issue with upgrading Amber by brew from 0.11.1 to 0.27.0 version: 

f ____$ brew upgrade amber
==> Upgrading 1 outdated package:
amberframework/amber/amber 0.11.1 -> 0.27.0
==> Upgrading amberframework/amber/amber 
==> Downloading https://github.com/amberframework/amber/archive/v0.27.0.tar.gz
Already downloaded: /Users/____/Library/Caches/Homebrew/downloads/87abc9b212e2021edb5680c7bdaa5694e4eb6efeae7993d54eabf481111af140--amber-0.27.0.tar.gz
==> shards install
==> make install PREFIX=/usr/local/Cellar/amber/0.27.0
Last 15 lines from /Users/____/Library/Logs/Homebrew/amber/02.make:
                ^~~~~~~~~~~~~~~~~~

in src/amber/support/message_encryptor.cr:23: instantiating "sign_bytes(Slice(UInt8))"

      if Crypto::Subtle.constant_time_compare(sign_bytes(data_iv), signature)
                                              ^~~~~~~~~~

in src/amber/support/message_encryptor.cr:66: no overload matches "OpenSSL::HMAC.digest" with types Symbol, String, Slice(UInt8)
Overloads are:
 - OpenSSL::HMAC.digest(algorithm : OpenSSL::Algorithm, key, data)

      OpenSSL::HMAC.digest(@digest, @secret, data)
                    ^~~~~~

make: *** [bin/amber] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/amberframework/homebrew-amber/issues

**Comment by elorest** [comment]
What version of crystal are you using?

**Comment by elorest** [comment]
0.27.0 works with crystal 0.27.x. If you try to compile it with 0.28.0 it will fail. Amber 0.28.0 will be released soon.

https://github.com/amberframework/amber/pull/1091/files

**Comment by zubroid** [comment]
> 0.27.0 works with crystal 0.27.x. If you try to compile it with 0.28.0 it will fail. Amber 0.28.0 will be released soon.
> 
> https://github.com/amberframework/amber/pull/1091/files

Thank you! Yes, I have Crystal 0.28.0 (2019-04-24)

**Comment by lxxxvi** [comment]
> 0.27.0 works with crystal 0.27.x. If you try to compile it with 0.28.0 it will fail. Amber 0.28.0 will be released soon.
> 
> https://github.com/amberframework/amber/pull/1091/files

I tried to to install amber `0.27.0` with crystal `0.27.(0|1|2)`, but it didn't work either... not sure if I did it right though.

I installed the crystal versions from the `.tar.gz` files that are provided on their [releases page](https://github.com/crystal-lang/crystal/releases), linked the `bin/crystal` and then ran `brew install amber`. I always got the same result as here and in #3 

Edit: 

Is it possible that even though I installed crystal `0.27.x` by my own, `brew install amber` uses the crystal executable from Homebrew?

MacOS 10.14.4 Mojave
